.github/workflows/buf-ci.yaml

```diff
@@ -0,0 +1,20 @@
+name: Buf CI
+on:
+  push:
+    paths:
+      - 'proto/**'
+permissions:
+  contents: read
```

Comment:
The permissions for comment writing is missing, intentional?

Reply:
Not intentional, I'll add it, thx